ezbake/config/logback.xml

```diff
@@ -18,6 +18,7 @@
 
     <root level="info">
         <!--<appender-ref ref="STDOUT"/>-->
+        <appender-ref ref="${logappender:-DUMMY}"/>
```

Comment:
FYI the original work was done here: https://github.com/puppetlabs/ezbake/pull/148